src/main/java/org/onebusaway/alexa/config/HandlerConfig.java

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright 2016 Sean J. Barbeau (sjbarbeau@gmail.com),
+ * Philip M. White (philip@mailworks.org)
```

Comment:
Assuming Amazon is signing the CLA as a corporation, can you please add a `, Amazon (your_email@amazon.com)`  to the copyright header after the existing authors?

Reply:
Yeah, I will add my own Email to it. but not end with amazon.